src/noverifier.rs

```diff
@@ -0,0 +1,18 @@
+use std::time::SystemTime;
+
+use rustls::client::{ServerCertVerified, ServerCertVerifier};
+
+pub struct NoVerifier;
+impl ServerCertVerifier for NoVerifier {
+    fn verify_server_cert(
+        &self,
+        _end_entity: &rustls::Certificate,
+        _intermediates: &[rustls::Certificate],
+        _server_name: &rustls::ServerName,
+        _scts: &mut dyn Iterator<Item = &[u8]>,
+        _ocsp_response: &[u8],
+        _now: SystemTime,
+    ) -> Result<ServerCertVerified, rustls::Error> {
+        Ok(ServerCertVerified::assertion())
```

Comment:
Its interesting that `rustls` has this not integrated by itself? Or is it a "if you really want to ignore everything, do it yourself" so you have to make more effort to get insecure stuff?

Reply:
Its intentional